src/main/java/ch/njol/skript/expressions/ExprExhaustion.java

```diff
@@ -0,0 +1,78 @@
+package ch.njol.skript.expressions;
+
+import javax.annotation.Nullable;
+
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+
+import ch.njol.skript.classes.Changer;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.PropertyExpression;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.util.SimpleExpression;
+import ch.njol.util.Kleenean;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Player Exhaustion")
```

Comment:
Remove the "Player"

Reply:
It can only be applied to players...

Reply:
exactly, thats why it's redundant

Reply:
All the other Name annotations doesn't use that

Reply:
Well, the syntax tells that it does not work on mobs. Besides, mobs do not get hungry. So it is a bit redundant information.